.github/workflows/ci.yml

```diff
@@ -1,288 +1,283 @@
 on:
```

Comment:
giga nit but pls configure your editor to not change the formatting on these files :D or let's do it in a separate PR intentionally

Reply:
idk why it formats on search and replace, that was unintentional